test.js

```diff
@@ -1,14 +1,38 @@
+import Path from 'path';
```

Comment:
```suggestion
import path from 'path';
```

Reply:
I actually do this as a convention to make `Path` like a static class/namespace, because often I need `path` as a variable name and that would overwrite the import in that scope.

But not a problem, I'll change it.